content/docs/alerting/configuration.md

```diff
@@ -64,12 +64,17 @@ global:
   # The default SMTP From header field.
   [ smtp_from: <tmpl_string> ]
   # The default SMTP smarthost used for sending emails.
+  # Provide the correct port for the smarthost (25 / 587 for SMTP over TLS (STARTTLS), 465 for deprecated SMTPS). 
```

Comment:
SMTPS is also SMTP over TLS, so this is a bit confusing.

Reply:
I am not sure how to properly write this. What do you think about removing the mentioning of `SMPTS` completely, because its actually not really in use anymore?

Reply:
If it's not what the user is after, there's no real need to mention it

Reply:
Removed. :)